mailet/standard/src/main/java/org/apache/james/transport/mailets/StripAttachment.java

```diff
@@ -361,11 +362,19 @@ private void addPartContent(BodyPart bodyPart, Mail mail, String fileName) throw
             fileNamesToPartContent = new LinkedHashMap<String, byte[]>();
             mail.setAttribute(attributeName, (Serializable) fileNamesToPartContent);
         }
-        ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream(bodyPart.getSize());
+        ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream(getSize(bodyPart));
```

Comment:
Test for getSize? Or tests for the wall class?

The wall is unit tested, and I actually broke some tests with the previous changeset. Hense this fix, solving it.